zebra-network/src/protocol/external/codec.rs

```diff
@@ -649,7 +650,7 @@ mod tests {
         let services = PeerServices::NODE_NETWORK;
         let timestamp = Utc.timestamp(1568000000, 0);
 
-        let rt = Runtime::new().unwrap();
+        let mut rt = Runtime::new().unwrap();
```

Comment:
The tokio runtime doesn't use interior mutability any more so it has to be mutable to spawn tasks.